swagger.yaml

```diff
@@ -101,6 +101,46 @@ paths:
           description: "Unexpected internal error"
           schema:
             $ref: '#/definitions/TokensErrorResponse'
+  /users:
```

Comment:
I know this is existing code but I think something similar to the following needs to be added to line 91 (above here) otherwise an error occurs in the swagger editor:

type: string

so that it looks like this:

parameters:
        - in: header
          type: string
          name: Authorization
          description: "The users access token as bearer token"

Reply:
done.